apps/web/src/components/commons/Toast/index.stories.tsx

```
@@ -0,0 +1,65 @@
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Success: Story = {
+  args: {
+    type: "success",
+    message: "자리 예약 성공!",
+  },
+};
+
+export const Error: Story = {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)